engine/src/main/java/com/redhat/cloud/notifications/events/EventConsumer.java

```diff
@@ -76,6 +77,7 @@ public void init() {
     @Incoming(INGRESS_CHANNEL)
     @Acknowledgment(PRE_PROCESSING)
     @Blocking
+    @ActivateRequestContext
```

Comment:
The reason why we need to add this annotation is explained in the [Quarkus migration guide for 2.13](https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.13#smallrye-reactive-messaging). In particular, we need the request context for all calls to RHOSE because we rely on a `BridgeAuth` bean that is produced with a `@RequestScoped` CDI scope.